packages/signature-controller/src/utils/decoding-api.ts

```diff
@@ -0,0 +1,61 @@
+import { EthMethod } from '@metamask/keyring-api';
```

Comment:
If we can't source that constant without this value, then better we define it ourselves to minimise dependencies.

Reply:
done